hnn_core/viz.py

```diff
@@ -82,6 +96,15 @@ def plot_dipole(dpl, tmin=None, tmax=None, ax=None, layer='agg', decim=None,
         The SciPy function :func:`~scipy.signal.decimate` is used, which
         recommends values <13. To achieve higher decimation factors, a list of
         ints can be provided. These are applied successively.
+    units : str | None
+        The physical units of the data, used for axis label. Defaults to
+        ``units='nAm'``. Passing ``None`` results in units being omitted from
+        plot.
```

Comment:
always use nAm (I would have preferred Am but that won't be backwards compatible I guess) for units? less knobs the better